tools/sass_generate_binaries.bzl

```diff
@@ -0,0 +1,17 @@
+load("@io_bazel_rules_sass//:defs.bzl", "sass_binary")
+
+# Generates multiple sass binaries based on a specified list of source files.
+# All generated sass binaries will be exposed as a filegroup that has all the
+# CSS outputs from the specified source files.
+def sass_generate_binaries(filegroup_name, source_files, sass_deps = []):
```

Comment:
This is unused

Reply:
It's unused now, but really needs to be used at some point in the demo app (will discuss this offline).

Reply:
"at some point" is the thing. Why not just add it then? This makes it hard to review, adds unnecessary bloat, and causes loss of context for the actual PR that uses this.

Reply:
Alright, but this is definitely going to need to be addressed with some immediacy.